R/server.R

```diff
@@ -161,7 +161,7 @@ createAppHandlers <- function(httpHandlers, serverFuncSource) {
   # This value, if non-NULL, must be present on all HTTP and WebSocket
   # requests as the Shiny-Shared-Secret header or else access will be
   # denied (403 response for HTTP, and instant close for websocket).
-  sharedSecret <- getOption('shiny.sharedSecret')
+  sharedSecret <- loadSharedSecret()
```

Comment:
A name like `checkSharedSecret` (as is used in the tests) would be a little clearer.